### PR TITLE
Update migrate-from-mysql-aurora.md

### PR DESCRIPTION
**Pull request by xxj123go**
树形目录标题和内容页标题不一致

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)
- [x] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts after cherry-picked

### CODE REVIEW

**Comment by yikeke** [comment]
Please help cherry-pick this PR to the master branch. @CharLotteiu

File: zh/migrate-from-mysql-aurora.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
-title: 从 MySQL 迁移数据 —— 以 Amazon Aurora MySQL 为例
+title: 从 Aurora 迁移到 TiDB
```
Comment (yikeke) [inline zh/migrate-from-mysql-aurora.md:2]: 受限于左侧目录树的空间，目录树的标题和原文标题并不一定一致。 @xxj123go 

这里的话，可以尽量保持一致：
```suggestion
title: 从 Aurora MySQL 迁移到 TiDB
```

Comment (yikeke) [reply zh/migrate-from-mysql-aurora.md:2]: @lance6716 PTAL

Comment (lance6716) [reply zh/migrate-from-mysql-aurora.md:2]: 确实 Aurora MySQL 好一些，因为还有 Aurora postgresql

Comment (lance6716) [reply zh/migrate-from-mysql-aurora.md:2]: 这里其实是想表达“从 MySQL 兼容的数据库”，比如各种云数据库，所以标题应该体现一下（原标题也不太好）

Comment (yikeke) [reply zh/migrate-from-mysql-aurora.md:2]:
```suggestion
title: 从兼容 MySQL 的数据库迁移数据 —— 以 Amazon Aurora MySQL 为例
```
要不然把文档标题改成上面这个⬆️

目录树标题改成“从兼容 MySQL 的数据库迁移到 TiDB”？ @lance6716 @xxj123go

Comment (yikeke) [reply zh/migrate-from-mysql-aurora.md:2]: @CharLotteiu 麻烦帮忙先按照上面这个建议改一下，如果有 comment 各位请再留

**Review by yikeke — APPROVED** [review]
LGTM

**Pull request merged**